doc/api/http2.md

```diff
@@ -2680,12 +2679,13 @@ if the stream is closed.
 [`EventEmitter`]: events.html#events_class_eventemitter
 [`Http2Stream`]: #http2_class_http2stream
 [`ServerHttp2Stream`]: #http2_class_serverhttp2stream
-[`ServerRequest`]: #http2_class_server_request
+[`ServerRequest`]: #http2_class_http2_http2serverrequest
```

Comment:
nit: `Http2ServerRequest`. You'll have to change its usage higher in the file.

Reply:
Done.